.github/ISSUE_TEMPLATE/ixd_user_story.md

```diff
@@ -0,0 +1,36 @@
+---
+name: 🧑‍🎨 IxD User story
```

Comment:
We can't use the same emoji for interaction design as we did for visual design! My suggestion is for a boring office worker instead but you or @leslieab has free reign to pick something else. 🧑‍🎤 and 🤦 were leading contenders for me.

```suggestion
name: 🧑‍💼 IxD User story
```